bin/build-l10n

```diff
@@ -7,7 +7,7 @@ import shutil
 import subprocess
 
 
-def main():
+def _main() -> None:
```

Comment:
Wreid, you want the main function to be protected, or you need the main symbol for something else ?